plugins/labhub.py

```diff
@@ -204,3 +204,76 @@ def mark_cmd(self, msg, match):
                         'so you will get feedback from the community. Use '
                         '`cobot mark wip` if there are known issues that should'
                         ' be corrected by the author.'.format(mr_link=mr.url))
+
+    @re_botcmd(pattern=r'^assign\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/issues/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
+               flags=re.IGNORECASE)
+    def assign_cmd(self, msg, match):
+        """Assign to GitLab and GitHub issues."""  # Ignore QuotesBear
+        org = match.group(2)
+        repo_name = match.group(3)
+        iss_number = match.group(4)
+
+        user = msg.frm.nick
+
+        try:
+            assert org == self.GH_ORG_NAME or org == self.GL_ORG_NAME
+        except AssertionError:
+            yield 'Repository not owned by our org.'
+            return
+
+        checks = []
+
+        def register_check(function):
+            checks.append(function)
+            return function
+
+        @register_check
+        def newcomer_difficulty_level(user, iss):
+            """
+            True if:
+            1. A newcomer is asking for assignment to low or newcomer issue.
+            2. The user is not a newcomer.
+            False if
+            1. A newcomer asks for assignment to an issue that has no difficulty
+               label.
+            2. A newcomer asks for assignment to an issue with difficulty higher
+               than low.
+            """
+            if self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user):
+                diff_labels = filter(lambda x: 'difficulty' in x, iss.labels)
+                if list(filter(lambda x: ('low' in x) or ('newcomer' in x),
+                               diff_labels)):
+                    return True
+                else:
+                    return False
+            else:
```

Comment:
Umm wouldn't this condition allow a user who is not even a newcomer to assign to issues? Like doesn't belong to any team?